docker/docker-entrypoint

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# WARNING: This file was auto-generated. Do not edit!
+#
+set -e
+
+case "$1" in
+notqmail)
+	if [ ! -f /etc/mtab ] ; then
+		if [ -f /proc/self/mounts ] ; then
+			echo "Warning  linking /etc/mtab to /proc/self/mounts" 1>&2
+			ln -s /proc/self/mounts /etc/mtab
+		elif [ -f /proc/mounts ] ; then
+			echo "Warning  linking /etc/mtab to /proc/mounts" 1>&2
+			ln -s /proc/mounts /etc/mtab
+		else
+			echo "Warning /etc/mtab: No such file or directory" 1>&2
+		fi
+	fi
```

Comment:
simply curious here: is that a boilerplate to get a full filesystem or is it specific to notqmail?

Reply:
Not specific to notqmail. I have a virtualdomains package which uses getmntent() call to find out the filesystem in which the user's Maildir is getting created (when adding a new user). It then creates maildirsize file for quota calculation. I guess the same can be done by using statvfs() call, but I have not yet investigated. This error came up (IIRC) on the ubuntu images. Most podman images are having the /etc/mtab file. The mounted filesystems are also in /proc/self/mounts. So I created this link so that user creation could go on without errors.

Reply:
Once you create this link, getmntent() library call doesn't fail. I believe getmntent() should be modified to not look at a hardcoded /etc/mtab file.

Reply:
Thank you for the insight. Maybe we adding a short comment like `# in case getmntent(3) is needed` before ?

getmntent(3):
> DESCRIPTION
>       These routines are used to access the filesystem description file
>       /etc/fstab and the mounted filesystem description file /etc/mtab.